project-codeflare/instascale
# Ensure deployment of configmap, and ocm-secret permissions.

# Issue link

#138

# What changes have been made

This pr contains adjustments which deploy necessary configurations for the operation of instascale, including the configmap and instascale-ocm-secret resource permissions.

# Verification steps

From an old branch:
1. Run make deploy to deploy Instascale.
1. Add your instascale-ocm-secret
1. Check the pod logs, and there will be a message stating the configmap has not been deployed.
1. Deploy the Instascale configmap independently.
1. Instascale will be unable to find the OCM secret and will automatically switch to machinesets. Even if an ocm secret is added.
1. Run `make undeploy`

From this branch:
1. Run make deploy to deploy Instascale.
1. Add your instascale-ocm-secret.
1. Check the pod logs, and the configmap and OCM secret should be found correcly.



## Checks
- Testing Strategy
